subjects/Microchip/C18/PIC18Extd/subject.cmd

```diff
@@ -0,0 +1 @@
+decompile.exe --heuristic shingle --arch pic --arch_option model="PIC18F1230" --loader IntelHex --base 0000 --entry 0000 PIC18EggExtd.hex
```

Comment:
Should be `--arch-option` instead of `--arch_option`

Reply:
Shall change this misspelled argument. 
No default model yet. Can think of it - likewise what is defined by Microchip for their COFF format (namely PIC18F4620).